hyperjumptech/monika
# Modify alert message to be more clear and send message as text

# Monika PR
## related issue #63 : Change notification in smtp to text and time to be known

## What issue this PR is trying to solve
There are some notification issue need to be solved :
1. Currently the smtp notification set as html, therefore there is no formatting and become one line.
2. The time in notification is sent as unix format, not readable to human.
3. need to define the messaging for each of the alerts

## How this PR solve the issue
1. set smtp send message as text
2. change unix time format to be sent as readable time in UTC
3. define message as describe in issue #63 

## How to test
1. set smtp notifications in config
2. set probe test in config
3. run monika, and wait for the alerts to be sent to through smtp
4. the message should be shown as defined in issue #63 

e.g :
Subject : [INCIDENT] Target https://github.com takes long to respond
Body :
      Target https://github.com is not healthy. The response time has been greater than 400 ms 3 times in a row
      
      Time: Wed, 24 Mar 2021 05:32:06 GMT
      Target URL: https://github.com
      From server: 172.17.2.228
